cli.ts

```diff
@@ -430,59 +426,61 @@ const initProject = (
 		map(_ => i18n.__('bootstrapMsg')),
 	);
 
+const runScaffoldPostInit = (scaffoldDir: SanitizedAbsPath.t): Future<TaqError.t, SanitizedAbsPath.t> => {
+	const scaffoldConfigAbspath = SanitizedAbsPath.create(`${scaffoldDir}/scaffold.json`);
+	return pipe(
+		readJsonFile<ScaffoldConfig.t>(scaffoldConfigAbspath),
+		map(logInput('Scaffold Config')),
+		coalesce(_ => ({}) as ScaffoldConfig.t)(identity),
```

Comment:
So if there's a read failure we just return the empty JSON I assume?

Reply:
Why not reject it instead?

Reply:
Correct. We don't reject because it's not a hard failure if the file doesn't exist. The fewer code branches (if/then) the less likelihood of a logic error, as there's only so many paths can the code can take.